src/cart/tests_common.h

```diff
@@ -104,6 +108,35 @@ tc_progress_stop(void)
 	opts.shutdown = 1;
 }
 
+/* Write a completion file to signal graceful server shutdown */
+void
+write_completion_file(void)
+{
+	FILE	*fptr;
+	char	*dir;
+	char	*completion_file;
+	char	*tmp_str;
+	char	pid[6];
+	pid_t	_pid;
+
+	_pid = getpid();
+	sprintf(pid, "%d", _pid);
+
+	dir = getenv("DAOS_TEST_SHARED_DIR");
+	D_ASSERTF(dir != NULL,
+        "DAOS_TEST_SHARED_DIR must be set for --write_completion_file "
```

Comment:
(style)  code indent should use tabs where possible

Reply:
https://github.com/daos-stack/daos/pull/5759/commits/e3732162fe0314afcb04b903d01a06b51e1067a1